Formula/l/lla.rb

```diff
@@ -0,0 +1,45 @@
+class Lla < Formula
+  desc "High-performance, extensible alternative to ls"
+  homepage "https://github.com/triyanox/lla"
+  url "https://github.com/triyanox/lla/archive/refs/tags/v0.2.6.tar.gz"
+  sha256 "6fc63f98d4a1da4e6ec231ba17b91b5ca8e472d3aa33fe7ea368c58cd920bf90"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "lla")
+
+    (buildpath/"plugins").each_child do |plugin|
```

Comment:
Should we test some of these plugins? Ideally we should set this up so that they can just be called by name without invoking a path. Linking `lla` with `-rpath` might suffice for that, depending on how the plugins are loaded.

Reply:
yeah, I have tested the plugins in my local, but did not bother as I did not find a good way to manipulate the plugins location, I can try again.

Reply:
just updated the tests to include some plugin checks